.github/workflows/build-images-workflow-run.yml

```
@@ -196,6 +194,7 @@ jobs:
     needs: [cancel-workflow-runs]
     env:
       GITHUB_CONTEXT: ${{ toJson(github) }}
+      GITHUB_REGISTRY_SECRET: ${{ secrets.GITHUB_REGISTRY }}
```

Just tested this -- github don't let you create a secret that starts with `GITHUB_`

```suggestion
      GITHUB_REGISTRY_SECRET: ${{ secrets.OVERRIDE_GITHUB_REGISTRY }}
```

Or similar.

---

sure. No problem with that